docs/notebooks/analysis/nullpoint.ipynb

```diff
@@ -0,0 +1,235 @@
+    "On the other hand, `null_point_find()` is used when the user does not have an adequate modeling function. It also does not require the grid to have uniform spacing in each dimension. Instead, it will ask the user for three arrays (one for each dimension) of coordinates that determines the desired custom spacing in each of the three dimensions, and then constructs the resulting grid on its owns. Furthumore, it requires the user to input all of the three components of magnetic field values, each as a 3D array with the same size as the grid, with each entry representing the strength of that component of the magnetic field for the corresponding grid coordinate. \n",
+    "Finally, both functions take in as arguments two converges threashholds that we will discuss later, for locating the null points. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72333e74-442c-4f7a-a3c9-76fad88eaac2",
+   "metadata": {},
+   "source": [
+    "## Locating a null point\n",
+    "Locating a null point is done via the trilinear analysis method disscused in the paper [**Hynes:2007**](https://docs.plasmapy.org/en/latest/bibliography.html#id20). There are three steps that goes into locating the null points of a given regular grid. \n",
+    "1) Reduction: First, every grid cells is checked for a simple condition so that we can rule out cells that can not contain a null point.\n",
+    "2) Trilinear Analysis: Assuming a trilinear field, the cells that have passed the reduction check are then analyzed, so that that we can be sure if they do contain a null point. \n",
+    "3) Locating the null point: The cell that contains a null point is isolated, and the location of the null point is estimated using the iterative Newton-Raphson method with an initial random guess."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2bd7ab2d-427f-4d85-9d58-352aeffc0b52",
+   "metadata": {},
+   "source": [
+    "## Classifying a null point\n",
+    "Classification is done by analysing the jacobian matrix calculated at the location of null point. The full method is explained in the paper by Hynes."
```

Comment:
Should this link be updated to [Parnell et al. (1996)](https://aip.scitation.org/doi/10.1063/1.871810)?